apps/calendar/js/app.js

```diff
@@ -259,12 +277,17 @@ module.exports = {
 
     this._routes();
 
-    var recurringEventsController = new RecurringEventsController(this);
-    this.observePendingObject(recurringEventsController);
-    recurringEventsController.observe();
-    this.recurringEventsController = recurringEventsController;
-
     nextTick(() => this.view('Errors'));
+
+    // Restart the calendar when the timezone changes.
```

Comment:
makes more sense here since this is only needed after the app is initialized. it also made it easier to measure how longs things were taking and to toggle the behavior.